.github/workflows/pkgci_regression_test_amdgpu_rocm.yml

```diff
@@ -48,5 +48,5 @@ jobs:
           source $VENV_DIR/bin/activate
           export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/lib:/opt/rocm/hip/lib
           pytest \
-            -s -m "plat_rdna3_rocm and presubmit" \
+            -rA -s -m "plat_rdna3_rocm and presubmit" \
             experimental/regression_suite
```

Comment:
Sample output: https://github.com/openxla/iree/actions/runs/7908294914/job/21587395578?pr=16415#step:6:80

```
==================================== PASSES ====================================
=========================== short test summary info ============================
PASSED experimental/regression_suite/tests/pregenerated/test_llama2.py::test_step_rdna3_rocm_stripped
======================= 1 passed, 9 deselected in 15.96s =======================
```

Yes - that is only running one test. As it turns out, https://github.com/openxla/iree/blob/main/experimental/regression_suite/tests/pregenerated/test_ukernel.py is only running on `plat_gfx90a_rocm` and `plat_gfx940_rocm`, not `plat_rdna3_rocm` 🤔

Reply:
fyi @raikonenfnu